engines/cruise/cruise.cpp

```diff
@@ -95,7 +95,9 @@ Common::Error CruiseEngine::run() {
 		ttsMan->enable(ConfMan.getBool("tts_enabled"));
 		ttsMan->setLanguage(ConfMan.get("language"));
 
-		if (getLanguage() == Common::RU_RUS) {
+		if (getLanguage() == Common::FR_FRA || getLanguage() == Common::IT_ITA) {
+			_ttsTextEncoding = Common::CodePage::kISO8859_1;
```

Comment:
```suggestion
			_ttsTextEncoding = Common::CodePage::kWindows1252;
```
I don't know if this is actually needed, but that should do no harm. Windows-1252 is an extension of Latin1, adding a few printable characters in places where Latin1 has non-printable characters. For French that would notably be the `œ`  missing in Latin1, but used for example in `cœur` (heart or core), `sœur` (sister), or `œuf` (egg), all quite common words that could be used in the game and may use that character (although they may also be spell with `oe` in which case both Latin1  and Windows-1252 would work).